docker-compose.yml.BACKUP

```diff
@@ -0,0 +1,122 @@
+            - DATABASE_NAME=slu-status
+        depends_on:
+            - mongo
+
+    one-shot-device:
+        build:
+            context: ./one-shot-device
+            dockerfile: Dockerfile
+        restart: always
+        labels:
+            - "traefik.enable=true"
+            - "traefik.http.routers.one-shot-device.entrypoints=web"
+            - "traefik.http.routers.one-shot-device.rule=PathPrefix(`/api/v1/one-shot-device`) || PathPrefix(`/api/v1/creator-devices`)"
+            - "traefik.http.services.one-shot-device.loadbalancer.server.port=3000"
+        environment:
+            - MONGO_URL=mongodb://${MONGO_INITDB_ROOT_USERNAME}:${MONGO_INITDB_ROOT_PASSWORD}@mongo-cxc:27017
+            - DB_NAME=one-shot-device
+            - IS_API_KEY=${API_KEY}
+            - IS_API_URL=${INTEGRATION_SERVICE}
+            - SLU_STATUS_BASE_URL=http://slu-status:3000/api/v1
+            - SLU_STATUS_API_KEY=${SLU_STATUS_API_KEY}
+        depends_on:
+            - mongo
+            - slu-status
+
+    authenticity-collector:
+        build:
+            context: ./authenticity-collector
+            dockerfile: Dockerfile
+        restart: always
+        labels:
+            - "traefik.enable=true"
+            - "traefik.http.routers.authenticity-collector.entrypoints=web"
+            - "traefik.http.routers.authenticity-collector.rule=PathPrefix(`/api/v1/authenticity`)"
+            - "traefik.http.services.authenticity-collector.loadbalancer.server.port=3000"
+        environment:
+            - API_KEY=${SLU_STATUS_API_KEY}
+            - DATABASE_URL=mongodb://${MONGO_INITDB_ROOT_USERNAME}:${MONGO_INITDB_ROOT_PASSWORD}@mongo-cxc:27017
+            - PROVE_OF_OWNERSHIP_URL=http://ssi-proof-of-ownership:3000/api/v1/ownership/prove
+            - IS_API_KEY=${API_KEY}
+            - IS_API_URL=${INTEGRATION_SERVICE}
+            - COLLECTOR_DID=${COLLECTOR_DID}
+            - COLLECTOR_SECRET=${COLLECTOR_SECRET}
+            - AUTH_PROVE_EXPIRATION=120000
+            - JWT_SECRET=${JWT_SECRET}
+        depends_on:
+            - ssi-proof-of-ownership
+            - mongo
+
+    dashboard:
+        build:
+            context: ./dashboard
+            dockerfile: Dockerfile
+        restart: always
+        labels:
+            - "traefik.enable=true"
+            - "traefik.http.routers.dashboard.entrypoints=web"
+            - "traefik.http.routers.dashboard.rule=PathPrefix(`/`)"
+            - "traefik.http.services.dashboard.loadbalancer.server.port=3000"
+        environment:
+            - VITE_IOTA_IS_SDK_API_KEY=${API_KEY}
+            - VITE_IOTA_IS_SDK_GATEWAY_URL=${INTEGRATION_SERVICE}
+            - VITE_DEVICE_MANAGEMENTE_API_REST_URL=
+            - VITE_SLU_STATUS_API_KEY=${SLU_STATUS_API_KEY}
+            - VITE_SLU_GATEWAY_URL=
+        depends_on:
+            - authenticity-collector
+            - one-shot-device
```

Comment:
I assume the docker-compose.yml.BACKUP is pushed by accident. If so delete it pls.

Reply:
Thanks, removed it from the remote cache